### PR TITLE
Translated "ASCII" to Ukrainian

### PR DESCRIPTION
**Pull request by Dod900ls1**
Fill in each of the sections (using NA for those that are not applicable).

## Author: 

- Yehor Boiar

## Language: 

- Ukrainian

## Terms defined:

- ASCII

### CODE REVIEW

**Review by olexandr-konovalov — CHANGES_REQUESTED** [review]
@Dod900ls1 Now this PR also includes commit from PR #658. This is not correct. When you create a feature branch, you had to first switch back to `main` by calling `git checkout main`, then pull changes from upstream (always a good idea), then call `git checkout -b ASCII` to create a new branch.

I suggest to leave this as is for now, and will fix this in another call. When you will create a new branch for another term, remember doing it from `main`.

See also the workflow in the contributor's guide to Glosario in its README: it uses web-based VSCode, but also first goes to `main` and creates a new branch off `main`.

**Comment by olexandr-konovalov** [comment]
@Dod900ls1 I've noticed now that you've edited your PR faster than I wrote my comment :) well done!

File: glossary.yml [hunk 1]
```diff
@@ -1238,6 +1238,13 @@
     def: >
       Njia ya kawaida ya kuwakilisha herufi zinazotumiwa sana katika lugha za Ulaya Magharibi kama nambari kamili 7- au 8-[bit](#bit), 
       ambayo sasa imechukuliwa na [Unicode](#unicode).
+  uk:
+    term: "ASCII"
+    def: >
+      Стандартний спосіб представлення символів, які зазвичай використовуються
+      в західноєвропейських мовах, у вигляді 7- або 8-бітних цілих чисел. 
+      Зараз замінений [Unicode](#unicode).
+
```
Comment (olexandr-konovalov) [inline glossary.yml:1247]: @Dod900ls1 another new line - not needed

**Review by Herobread — APPROVED** [review]

File: glossary.yml [hunk 2]
```diff
@@ -1238,6 +1238,12 @@
     def: >
       Njia ya kawaida ya kuwakilisha herufi zinazotumiwa sana katika lugha za Ulaya Magharibi kama nambari kamili 7- au 8-[bit](#bit), 
       ambayo sasa imechukuliwa na [Unicode](#unicode).
+  uk:
+    term: "ASCII"
+    def: >
+      Стандартний спосіб представлення символів, які зазвичай використовуються
+      в західноєвропейських мовах, у вигляді 7- або 8-бітних цілих чисел. 
+      Зараз замінений [Unicode](#unicode).
```
[the remark this answers is not in the document]
Comment (Dod900ls1) [reply glossary.yml]: замінений на [Unicode] відчуваєтся більш природньо. Виправив на цей варіант

**Review by olexandr-konovalov — APPROVED** [review]
@elletjies please merge

**Pull request merged**